presto-main/src/main/java/io/prestosql/security/ViewAccessControl.java

```diff
@@ -74,4 +78,14 @@ public void checkCanGrantExecuteFunctionPrivilege(SecurityContext context, Strin
     {
         return delegate.getColumnMasks(context, tableName, columnName, type);
     }
+
+    private void executeAsViewOwner(Consumer<AccessControl> consumer)
+    {
+        try {
+            consumer.accept(delegate);
+        }
+        catch (AccessDeniedException e) {
+            throw new PrestoException(PERMISSION_DENIED, "View owner does not have sufficient privileges: " + e.getMessage(), e);
```

Comment:
Please throw `io.prestosql.spi.security.AccessDeniedException`

Reply:
I think that AccessDeniedException 
will be inconsistent with this conversation: https://github.com/prestosql/presto/pull/5086#discussion_r489786857
and relates a little bit to https://github.com/prestosql/presto/pull/5086#discussion_r486670501

Reply:
Thanks